fragment-profile.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <title>Gallery of Glosses</title>
+    <link rel="stylesheet" href="css/gloss.css">
+    <link href="favicon.ico" rel="icon" type="image/x-icon" />
+    <script src="js/auth.js" type="module"></script>
+    <script src="js/deer.js" type="module"></script>
+    <script src="js/layout.js" type="module"></script>
+    <!-- <script src="js/text-witness-line-selector.js" type="module"></script> -->
+    <script src="js/bibliographicCitationView.js" type="module"></script>
+    <script src="js/shared.js"></script>
+    <style>
+        input[disabled],select[disabled],textarea[disabled]{
+            border: none;
+            background-color: transparent;
+            padding: 0;
+            display: inline;
+            font-size: 1em;
+            opacity: 1;
+        }
+        .pageContainer.is-toggled{
+            visibility: hidden;
+            height: 0px;
+        }
+        .loadingMsg{
+            color: white;
+            padding-top: 4em;
+        }
+        line:nth-child(2n) {
+            background-color: unset;
+        }
+        nav[user-input]{
+            margin-bottom: 1em;
+        }
+        nav[user-input] a{
+            cursor: pointer;
+        }
+        #reset{
+            cursor: pointer;
+        }
+        gloss-modal-button{
+            display: none;
+        }
+        .toggleInclusion.primary{
+            display: none;
+        }
+        #openCitationModalButton{
+            display: none;
+        }
+    </style>
+</head>
+
+<body class="container">
+    <gog-header class="container"></gog-header>
+    <h2 class="bg-basic"> Witness Fragment Profile </h2>
+    <p id="look" class="bg-light">
+        Witness Fragment Details
+    </p>
+    <!-- Perhaps this could be a Custom Element widget of some kind -->
+    <div id="loading" class="is-hidden">
+        <p class="loadingMsg">
+            Please be patient while we get things together...
+        </p>
+    </div>
+    <bibliographic-citation-div class="entity-needed is-hidden"></bibliographic-citation-div>
+    <div class="redirectToWitness bg-light entity-needed is-hidden">
+        <h4> Modify Fragment Metadata </h4>
+        <p> <u>Redirect</u> to the Fragment Metadata page by clicking below. </p>
+        <a class="addWitnessBtn button secondary" onclick="metadataForWitness(true)" style="margin: 5px; padding: 10px; border: none; border-radius: 4px;">Redirect to Modify Fragment Metadata</a>
+    </div>
+    <div class="referencedGlossContainer bg-light entity-needed is-hidden">
+        <h4> Referenced Gloss </h4>
+        <deer-view class="" id="referencedGloss" deer-template="label">
+            Loading&hellip;
+        </deer-view>
+    </div>
+    <form id="witnessForm" class="entity-needed is-hidden bg-basic" deer-type="Text" deer-context="http://purl.org/dc/terms">
```

Comment:
my branch had this turned to not a form, which I think is better. I'm not so committed that I cannot accept this.

Reply:
That is because our "detail" page for our entities is a populated form.  I stuck with that mechanic.